source/blog/2017-02-06-finagle-loves-netty4.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Netty 4 support in Finagle
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Finagle, Netty4
+---
+
+It's been quite a while since [Netty 4 migration in Finagle][netty4-blog-post] was initially
+announced. We've travelled a long way and are happy to announce that there is now (as per
+[Finagle 6.42][6.42]) a support for Netty 4 transports in most of the protocols: Thrift, ThriftMux,
+Memcached, MySQL, Kestrel, and Redis.
+
+## How to enable Netty 4?
+
+While we have not yet defaulted to Netty 4, we've been running it in production for several months
+and gained enough confidence to publicize the availability of the alternative transports in Finagle.
+
+We encourage Finagle users to try out the new Netty 4 transports for their protocols and jump on a
+fast track to upcoming changes around resiliency (think of HTTP/2) and performance (think of Netty 4
```

Comment:
instead of saying Netty 4 pooling, how about describing a generic improvement instead?  that way people who don't work closely with netty know what we're talking about.